components/gitpod-cli/cmd/env.go

```diff
@@ -80,14 +89,14 @@ type connectToServerResult struct {
 	repositoryPattern string
 	wsInfo            *supervisorapi.WorkspaceInfoResponse
 	client            *serverapi.APIoverJSONRPC
-
-	useDeprecatedGetEnvVar bool
 }
 
 type connectToServerOptions struct {
 	supervisorClient *supervisor.SupervisorClient
 	wsInfo           *api.WorkspaceInfoResponse
 	log              *log.Entry
+
+	setEnvScopeUser bool
```

Comment:
I think this would be nicer if we just passed down the scope instead of this boolean, we can reuse the `envScope` type.